### DISCUSSION TITLE
Can I change the Port ?

### DISCUSSION THREAD

**Discussion by SujinIn** [rtCamp/Frappe-Manager#107] [General]
I Thank you for the 0.10.0 release.
The upgrade went smoothly, and the content is well preserved.

I've created the ERP as "erp" using "fm create."
Although I mentioned it in the issue, I can access `erp.localhost` from the server, but not from other computers (503 error).

So I'm considering trying Traefik.
In that case, I would like to use ports such as 80, 8080, and 443, which are currently used for erpnext in fm.
I just want to maintain the current situation where I can access `erp.localhost` and also enable access to `erp.com` or (only accessible within the company's internal network) `192.168.0.250 `using Traefik.
So, does anyone know how to modify the ports in fm?

Thank you very much.

**Comment by Xieyt** [comment]
Hi @SujinIn,

The reason behind this issue is the incorrect DNS resolution of `erp.localhost`. Currently, the DNS resolution is pointing to `127.0.0.1`, whereas it should be directed to the IP defined in the hosts file. This discrepancy arises from naming the site as a **subdomain** of **localhost** I'll file an issue regarding this and ensure that users aren't restricted to using only subdomains of localhost when creating site. Once this is addressed and defined in the hosts file, everything should function normally. Expect this fix to be available in the upcoming release.

**Reply by SujinIn** [reply]
Thank you. I was very excited to discover Frappe Manager and wanted to try ERPNext through an easy installation. Due to the problem you mentioned, I tried many things, such as building a reverse proxy or modifying the host, but since I am not an expert, it was still too difficult. Of course, I studied a lot :) 
Thanks for solving it.

**Reply by Xieyt** [reply]
Hey @SujinIn, Latest [release](https://github.com/rtCamp/Frappe-Manager/releases/tag/v0.11.0) contains the feature #113. Please use it and LMK if it works for you.
